mozilla/fx-private-relay
# Insert final plan IDs for new Premium and Bundle

These should be the final plan and product IDs for the new Premium
and Bundle subscriptions in production, as well as prices for the new Premium subscriptions.

![image](https://user-images.githubusercontent.com/4251/190157411-87ed3528-b186-47a3-97f9-6e6addfe3030.png)

The actual prices for phones and bundle, as well as the product and plan IDs for the
phone plans, are yet to come.

This also makes the plan ID for the bundle in the US and Canada
configurable via an environment variable, so we can test the
subscription flow on stage.

# How to test

At `api/v1/runtime_data`, the updated Plan IDs should be visible under PERIODICAL_PREMIUM_PLANS and BUNDLE_PLANS. Additionally, if you add the stage plan IDs to your `.env` as follows, you should see those under `BUNDLE_PLANS`:

```env
BUNDLE_PROD_ID=prod_MQ9Zf1cyI81XS2
BUNDLE_PLAN_ID_US=price_1LhJGWKb9q6OnNsLuCurigPk
```

# Checklist
